web/modules/custom/usa_translation/usa_translation.module

```diff
@@ -46,8 +46,9 @@ function usa_translation_page_attachments_alter(&$page) {
     }
 
     $entity = $route_match->getParameter($name);
-    // If Field Language toggle exist and is set then get it's node value.
-    if (method_exists($entity, 'getFields') != FALSE
+    // If Field Language toggle exist and is set then get its node value.
+    if ($entity // counterexample: type "entity:filter_format" does not produce an entity. Path /filter/tips
+      && method_exists($entity, 'getFields') != FALSE
```

Comment:
I think you could get more specific with a single test to replace both of these checks with one: `if ($entity instance of FieldableEntityInterface`. Doing so ensures you have an `Entity` with the `getFields()` method.

Reply:
Much better, thanks!